docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+ 0
+ 1
+ 2
+ 3
+ 4
+
+julia> @view german[2, 2]
+0-dimensional view(::Vector{Int64}, 2) with eltype Int64:
+22
+
+julia> @view german[3, 2:5] # a DataFrameRow, the same as for german[3, 2:5] without a view
+DataFrameRow
+ Row │ Age    Sex     Job    Housing
+     │ Int64  String  Int64  String
+─────┼───────────────────────────────
+   3 │    49  male        1  own
+
+julia> @view german[2:5, 2:5] # here you can see creation of view is very fast
+4×4 SubDataFrame
+ Row │ Age    Sex     Job    Housing
+     │ Int64  String  Int64  String
+─────┼───────────────────────────────
+   1 │    22  female      2  own
+   2 │    49  male        1  own
+   3 │    45  male        2  free
+   4 │    53  male        2  free
+```
+
+## Using `select`, and `select!`, `transform`, and `transform!`
+
+You can also use the `select` and `select!` functions to select, rename, and transform columns in a data frame.
+
+The `select` function creates a new data frame:
+
+```jldoctest dataframe
+julia> german = copy(german_ref);
+
+julia> select(german, Not(:Age)) # drop column :Age in a new data frame
+1000×9 DataFrame
+  Row │ id     Sex     Job    Housing  Saving accounts  Checking account  Cred ⋯
+      │ Int64  String  Int64  String   String           String            Int6 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0  male        2  own      NA               little                 ⋯
+    2 │     1  female      2  own      little           moderate
+    3 │     2  male        1  own      little           NA
+    4 │     3  male        2  free     little           little
+    5 │     4  male        2  free     little           little                 ⋯
+    6 │     5  male        1  free     NA               NA
+    7 │     6  male        2  own      quite rich       NA
+    8 │     7  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮       ⋮            ⋮                ⋮               ⋱
+  994 │   993  male        3  own      little           little                 ⋯
+  995 │   994  male        2  own      NA               NA
+  996 │   995  female      1  own      little           NA
+  997 │   996  male        3  own      little           little
+  998 │   997  male        2  own      little           NA                     ⋯
+  999 │   998  male        2  free     little           little
+ 1000 │   999  male        2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+
+julia> select(german, r"Sex") # select columns containing 'Sex' character
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+
+julia> select(german, r"Sex","Job") # Selects more than one column
```

Comment:
the above operation also could have selected more than one column

Reply:
Yeah, my intention was to select one column and then show how to select more than one column